docs/dev/table/catalog.md

```diff
@@ -179,10 +179,11 @@ Currently `HiveCatalog` supports most Flink data types with the following mappin
 | TIMESTAMP_WITHOUT_TIME_ZONE |  Timestamp |
 | TIMESTAMP_WITH_TIME_ZONE |  N/A |
 | TIMESTAMP_WITH_LOCAL_TIME_ZONE |  N/A |
-| INTERVAL |  N/A |
-| BINARY        |  binary |
-| VARBINARY(p)  |  binary |
-| ARRAY\<E>     |  list\<E> |
+| INTERVAL      |   N/A*** |
+| BINARY        |   N/A |
+| VARBINARY(p)  |   N/A |
+| BYTES         |   BINARY |
+| ARRAY\<E>     |  ARRAY\<E> |
 | MAP<K, V>     |  map<K, V> |
```

Comment:
Let's add a note saying that Hive map key type only allows primitive types, while Flink map key can be any data type.

Reply:
+1

Reply:
Why not throw exception in `HiveTypeUtil.toHiveTypeInfo` when hive not support?

Reply:
> HiveTypeUtil.toHiveTypeInfo

Indeed we throw `UnsupportedOperationException` in `HiveTypeUtil.toHiveTypeInfo`.